.github/workflows/release.yml

```diff
@@ -0,0 +1,92 @@
+name: Release new version
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Release version (default: auto)"
+      preRelease:
+        description: "Is this a pre-release? (true/false, default: auto)"
+      dryRun:
+        description: Dry Run
+        default: false
+
+jobs:
+  release:
+    name: Build and publish a release
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/${{ github.repository_owner }}/xdg-desktop-portal-build:latest
+    permissions:
+      contents: write
+    steps:
+      - name: Configure git user
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --global --add safe.directory $GITHUB_WORKSPACE
```

Comment:
The username and email are required to have the commit appear like it was made by the official github-actions bot (the same one that creates releases, as you can see [here](https://github.com/flexagoon/xdg-desktop-portal/releases/tag/1.21.1) for example). AFAIK the email is not documented anywhere officially, I just found it on some random GitHub discussion page and it worked.

The safe.directory thing is needed because otherwise git is complaining about it and doesn't allow to add commits.